docs/getting-started-guides/ubuntu/local.md

```diff
@@ -13,48 +13,23 @@ The purpose of using [LXD](https://linuxcontainers.org/lxd/) on a local machine
 {% capture prerequisites %}
 ## Prerequisites
 
-In order to simplify local deployment this method leverages the [Conjure Up tool](
-http://conjure-up.io/). 
-
-This will provide a pseudo-graphical set up in a terminal that is simple enough for developers to use without having to learn the complexities of operating Kubernetes. This will enable new developers to get started with a working cluster.
-{% endcapture %}
-
-{% capture steps %}
-## Getting Started
-
-First, you need to configure LXD to be able to host a large number of containers. To do this we need to update the [kernel parameters for inotify](https://github.com/lxc/lxd/blob/master/doc/production-setup.md#etcsysctlconf).
-
-On your system open up `/etc/sysctl.conf` *(as root) and add the following lines:
-
-```
-    fs.inotify.max_user_instances = 1048576
-    fs.inotify.max_queued_events = 1048576
-    fs.inotify.max_user_watches = 1048576
-    vm.max_map_count = 262144
-```
-
-_Note: This step may become unnecessary in the future_
-
-Next, apply those kernel parameters (you should see the above options echoed back out to you):
-    
-    sudo sysctl -p
-   
-Now you're ready to install conjure-up and deploy Kubernetes.
+Install [conjure-up](http://conjure-up.io/), a tool for deploying big software.
     
 ```
     sudo snap install conjure-up --classic
-    conjure-up kubernetes
 ```
 
-Note: During this set up phase conjure-up will ask you to "Setup an ipv6 subnet" with LXD, ensure you answer NO. ipv6 with Juju/LXD is currently unsupported.
+Note: If conjure-up asks you to "Setup an ipv6 subnet" with LXD, answer NO. ipv6 with Juju/LXD is currently unsupported.
+{% endcapture %}
 
-### Walkthrough
+{% capture steps %}
+### Deploying Kubernetes
```

Comment:
This should be at the same heading level as *Overview* and *Before you begin*, so make it an h2 instead of h3, i.e. `## Deploying Kubernetes`.